src/sssom/obo.epm.json

```diff
@@ -735,8 +737,8 @@
             "https://identifiers.org/clo/",
             "https://identifiers.org/clo:",
             "https://purl.obolibrary.org/obo/CLO_",
-            "https://www.ebi.ac.uk/ols/ontologies/clo/terms?iri=http://purl.obolibrary.org/obo/",
             "https://www.ebi.ac.uk/ols/ontologies/clo/terms?iri=http://purl.obolibrary.org/obo/CLO_",
+            "https://www.ebi.ac.uk/ols4/ontologies/clo/classes?iri=http://purl.obolibrary.org/obo/",
```

Comment:
@cthoyt i didn’t notice this before but this seems a bogus prefix to me, dont you agree? The one before this one is correct though.